vendor/github.com/mongodb/go-client-mongodb-atlas/mongodbatlas/clusters.go

```diff
@@ -114,6 +114,61 @@ type clustersResponse struct {
 	TotalCount int       `json:"totalCount,omitempty"`
 }
 
+// DefaultDiskSizeGB represents the Tier and the default disk size for each one
+// it can be use like: DefaultDiskSizeGB["AWS"]["M10"]
+var DefaultDiskSizeGB map[string]map[string]float64 = map[string]map[string]float64{
```

Comment:
This will be fun to keep updated!! :)

Reply:
I will remove it due that it isn't already necessary in the Terraform part, I tested again without this change verifying if the MongoDB server set up correctly the  Disk Size by default, and everything is ok.